sdks/go/examples/wordcount/wordcount.go

```diff
@@ -111,9 +111,11 @@ func init() {
 }
 
 var (
-	wordRE  = regexp.MustCompile(`[a-zA-Z]+('[a-z])?`)
-	empty   = beam.NewCounter("extract", "emptyLines")
-	lineLen = beam.NewDistribution("extract", "lineLenDistro")
+	wordRE      = regexp.MustCompile(`[a-zA-Z]+('[a-z])?`)
+	empty       = beam.NewCounter("extract", "emptyLines")
+	minLength   = flag.Int("min_length", 9, "Minimum word length")
```

Comment:
Since this is a new flag, and this example doesn't filter words, we might want to just call it "small_word_length" instead, and clarify that it determines what gets counted.